core/domain/user_jobs_one_off.py

```diff
@@ -626,3 +628,63 @@ def map(item):
     def reduce(key, values):
         values.sort()
         yield (key, values)
+
+
+class VerifyProfilePictureOneOffJob(jobs.BaseMapReduceOneOffJobManager):
```

Comment:
Possible alternate name: ``ProfilePictureAuditOneOffJob``. I think we have more jobs with 'audit' included in their names for this type of purpose.